ej2-react/pdfviewer/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with Standalone React PDF Viewer component | Syncfusion
-description: Learn here all about Getting started with Standalone React PDF Viewer component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with Standalone React PDF Viewer | Syncfusion
+description: Learn here all about getting started with Standalone React PDF Viewer component of Syncfusion Essential JS 2 and more details.
 control: PDF Viewer
 platform: ej2-react
 documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Getting Started with Standalone PDF Viewer component
+# Getting Started with Standalone PDF Viewer Component
```

Comment:
Update platform name, what is the meaning of Standalone?

Reply:
Resolved